legion/engine/core/compute/context.hpp

```diff
@@ -65,6 +67,55 @@ class Context
         return Buffer(m_context,data,size,type,std::forward<std::string>(name));
     }
 
+    static Buffer createImage(image& img,buffer_type type, std::string name ="")
+    {
+        size_type width = img.size.x;
+        size_type height = img.size.y;
+        size_type depth = 0;
+
+        cl_image_format fmt;
+
+        switch(img.format){
+        case channel_format::eight_bit: fmt.image_channel_data_type = CL_UNORM_INT8; break;
+        case channel_format::sixteen_bit: fmt.image_channel_data_type = CL_UNORM_INT8; break;
```

Comment:
sixteen bit uint8?

Reply:
CTRL-C,CTRL-V is a dangerous drug man. resolved in 7563ccc